redbeat/schedulers.py

```diff
@@ -116,6 +117,11 @@ def get_redis(app=None):
                                 password=redis_options.get('password'),
                                 decode_responses=True)
             connection = sentinel.master_for(redis_options.get('service_name', 'master'))
+        elif conf.redis_url.startswith('rediss'):
+            ssl_options = { 'ssl_cert_reqs': ssl.CERT_REQUIRED }
```

Comment:
I think there is a valid use cases where a snakeoil cert is being used along with a ```rediss://``` URL, which means we can't blindly enforce CERT_REQUIRED. As per our other conversation, this feels wrong from a UX perspective but is in line with existing behaviour in other packages in the ecosystem.

Reply:
Ah, good point.  In that case, should I could raise an exception if the value of `redbeat_redis_use_ssl` is not a dictionary?

Reply:
@sibson What do you think? I think, given that the user can still override the value to `CERT_NONE` in the case they're using snakeoil certs, that the current implementation should be fine.  Happy to change it to something else if you think there's a better way.

Reply:
Thanks for pointing that out.  It's pretty reasonable stance to take.  My hesitation would be that it's a breaking backwards incompatible change. Being as it should fail early and spectacularly, I think we can include it as is with a version bump and clearly documented.  I'll merge as is once I check if we need a release of current master before doing the version bump.

Reply:
Ah, right, backwards compatibility!  Depending on how much work you want to do, we could release a version of this change that's backwards compatible but unsafe, having it log a warning saying the behavior will change in the major version, and then merge this PR as is on the major version.  Just waiting for the major version is fine too though.

Either way, thanks for taking a look and helping me flesh this out and getting it in =)

Reply:
I'm tagging this for 2.0 release, once I clear the decks on a 1.0 release I'll merge this.

Reply:
Cool, thanks for the update!